text/0000-fmt-debug-hex.md

```diff
@@ -0,0 +1,158 @@
+- Feature Name: fmt-debug-hex
+- Start Date: 2017-11-24
+- RFC PR:
+- Rust Issue:
+
+# Summary
+[summary]: #summary
+
+Add support for formatting integers as hexadecimal with the `fmt::Debug` trait,
+including when the occur within larger types.
+
+```rust
+println!("{:02X?}", b"AZaz\0")
+```
+```
+[41, 5A, 61, 7A, 00]
+```
+
+# Motivation
+[motivation]: #motivation
+
+Sometimes the bits that make up an integer are more meaningful than its purely numerical value.
+For example, an RGBA color encoded in `u32` with 8 bits per channel is easier to understand
+when shown as `00CC44FF` than `13387007`.
+
+The `std::fmt::UpperHex` and `std::fmt::LowerHex` provide hexadecimal formatting
```

Comment:
nit: The `std::fmt::UpperHex` and `std::fmt::LowerHex` _traits_?

Reply:
Fixed, thanks.